pkg/gitparse/gitparse.go

```diff
@@ -20,6 +20,9 @@ import (
 // DateFormat is the standard date format for git.
 const DateFormat = "Mon Jan 02 15:04:05 2006 -0700"
 
+// MaxDiffSize is the maximum size for a diff. Larger diffs will be cut off.
```

Comment:
nit: We can group the two consts together?

const (
    // DateFormat is the standard date format for git.
    DateFormat = "Mon Jan 02 15:04:05 2006 -0700"

    // MaxDiffSize is the maximum size for a diff. Larger diffs will be cut off.
    MaxDiffSize = 1 * 1024 * 1024 * 1024 // 1GB
)